package.json

```diff
@@ -83,31 +84,23 @@
   },
   "devDependencies": {
     "@testing-library/dom": "^10.4.0",
-    "@testing-library/jest-dom": "^6.1.5",
```

Comment:
Despite the name, I think we can keep `@testing-library/jest-dom`:

https://testing-library.com/docs/svelte-testing-library/setup/